pkg/dns/azure/client/auth.go

```diff
@@ -2,15 +2,17 @@ package client
 
 import (
 	"errors"
-	"os"
-	"strings"
-
+	"fmt"
```

Comment:
Mind bumping this up to the previous group?

Reply:
Yes, sorry, I'm not sure how those got moved around (assuming my IDE did it).